src/main/java/com/google/api/generator/engine/ast/JavaEscaper.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+import com.google.common.escape.Escaper;
+import com.google.common.escape.Escapers;
+
+public class JavaEscaper {
```

Comment:
This name is rather generic, can we use a more descriptive one?

Reply:
Also, if this will be used exclusively by StringObjectValue and no other class, consider making this a nested class in StringObjectValue.

Reply:
renamed to `StringValueEscaper` and put it in `escaper` package.

Reply:
As above: This seems to be used only by StringObjectValue, so it can be a nested class in there. What is the motivation for putting it in a separate package?